comictalker/__init__.py

```diff
@@ -30,7 +32,15 @@ def get_talkers(version: str, cache: pathlib.Path) -> dict[str, ComicTalker]:
             if obj.id != talker.name:
                 logger.error("Talker ID must be the same as the entry point name")
                 continue
-            talkers[talker.name] = obj
+            try:
+                if parse(version) >= parse(obj.comictagger_min_ver):
+                    talkers[talker.name] = obj
+                else:
+                    logger.error(
+                        f"CT minimum required version not met for talker: {talker.name} with version: {obj.comictagger_min_ver}"
+                    )
+            except InvalidVersion:
+                logger.error(f"Invalid version number for talker: {talker.name} - version: {obj.comictagger_min_ver}")
```

Comment:
I think this should be a warning instead and we should still try to use the plugin